Makefile

```diff
@@ -0,0 +1,55 @@
+.PHONY: clean start-ctr db ear image deploy-db deploy
```

Comment:
Nitpick:  I completely understand *why* this is being done this way, but for general uPortal acceptance, I'd love not to introduce yet another build system.  We already have ant, maven and gradle.  Adding make into the mix just feels gross.  I really hope the long term plan for uPortal is to move to 100% gradle.  This feels counter to that.